test/integration/route-href.spec.ts

```diff
@@ -1,11 +1,13 @@
-import {TemplatingRouteLoader} from 'src/route-loader';
-import {bootstrap} from 'aurelia-bootstrapper';
-import {RouteLoader, AppRouter, Router} from 'aurelia-router';
-import {StageComponent} from 'aurelia-testing';
+import { TemplatingRouteLoader } from '../../src/route-loader';
```

Comment:
what happened here? just more explicit? i tend to prefer using a rooted path to prevent ../ hell

Reply:
I need to validate / enhance test config to make sure root path works. It took me a lot of trial and errors to understand what part of the converted setup didn't work. When I restart the work on this PR, i'll review this

Reply:
I should point out here that in my experience writing and refactoring code/tests in vNext, I'm favoring ../ hell over rooted paths. Reason being that typescript and vs code are (currently) kind of bad at keeping the references linked. So you could refactor-rename something under src and it would not propagate to the tests. Very troublesome. With horrible deep ../ paths it does all work 100% reliably. Just food for thought.